edi_oca/static/src/xml/widget_edi.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <templates id="template" xml:space="preserve">
     <div t-name="edi_oca.EdiConfigurationWidget">
-        <t t-foreach="info" t-as="rule_id" t-key="rule_index">
```

Comment:
Also done in https://github.com/odoo/enterprise/blob/18.0/spreadsheet_edition/static/src/bundle/list/side_panels/edit_list_sorting_section/edit_list_sorting_section.xml#L15